### PR TITLE
feat(rest-api): update page

### PR DESCRIPTION
**Pull request by laurentleseigneur**
* new signature
* change titles

### CODE REVIEW

File: md/rest_extensions.md [hunk 1]
```diff
@@ -4,10 +4,13 @@
...
 API used by Operations Management Living Application
 
-
 ## `/bici-case-api`
-### `GET`
+#### Http method
+`GET`
+
+#### Description
 Get list of running cases
```
Comment (NathalieC) [inline md/rest_extensions.md]: "Get the list of open cases."
"open cases" is the term used in the case list. It's worth reusing everywhere.

File: md/rest_extensions.md [hunk 2]
```diff
@@ -19,18 +22,29 @@ Get list of running cases
 |search|string|false|Search keyword, will search in case id and name of the initiator|
 
 ## `/bici-process-api`
```
Comment (NathalieC) [inline md/rest_extensions.md:24]: in the table: "Page to return, starts at 0"
                    " late, predictedLate, onTime"

File: md/rest_extensions.md [hunk 3]
```diff
@@ -19,18 +22,29 @@ Get list of running cases
...
-### `GET`
+#### Http method
+`GET`
+
+#### Description
 Get processes
+
```
Comment (NathalieC) [inline md/rest_extensions.md:30]: A list of process? All processes?

File: md/rest_extensions.md [hunk 4]
```diff
@@ -19,18 +22,29 @@ Get list of running cases
...
+#### Description
 Execute a query
+
 #### Query parameters
 |Parameter|Type|Required|Description|
 |---------|----|--------|-----------|
 |c|integer|false|Number of elements to return|
 |p|integer|false|Page to return, start at 0|
+|queryName|string|false|Name of the analytic query |
```
Comment (NathalieC) [inline md/rest_extensions.md]: idem, "starts at 0"
           "analytics query". I don't think analytic is usable as singular.

File: md/rest_extensions.md [hunk 5]
```diff
@@ -19,18 +22,29 @@ Get list of running cases
...
+|processName|string|true|Name of the process|
+|processVersion|string|false|Version of the process. If empy, use all allowed versions|
```
Comment (NathalieC) [inline md/rest_extensions.md:48]: "If empty, it uses all versions allowed to the logged user"

File: md/rest_extensions.md [hunk 6]
```diff
@@ -19,18 +22,29 @@ Get list of running cases
...
 |parameters|object|false|Parameters required by the query|
```
Comment (NathalieC) [inline md/rest_extensions.md:49]: parameters is a parameter of the query?

**Review by NathalieC — CHANGES_REQUESTED** [review]

**Comment by laurentleseigneur** [comment]
http://doc.rd.lan:12039/bici/1.1/rest_extensions

File: md/rest_extensions.md (hunk 6 above)
Comment (laurentleseigneur) [reply md/rest_extensions.md:49]: it's a set of parameters, depends on the query

**Pull request merged**